scripts/justmat/paths.lua

```diff
@@ -209,17 +231,18 @@ end
 
 function key(n, z)
   if n == 1 and z == 1 then
-    alt = true
-  else
-    alt = false
+    time_last = util.time()
+    if util.time() - time_last <= 1.0 then
+      step_edit_mode = not step_edit_mode
+    end
   end
 
   if n == 2 and z == 1 then
     enc_sel_mode = true
   else
     enc_sel_mode = false
   end
-
+  
```

Comment:
aside: are you editing in maiden?

(i ask because automating whitespace cleanup is on my short list.)

Reply:
well maiden doesn't help where it should imo.  no-one should have to pay attention to unnecessary whitespace.  that's really the editor's job.

opened https://github.com/monome/maiden/issues/139 to track support for in-app formatting.